### PR TITLE
CI configuration for openshift/mdns-publisher

### PR DESCRIPTION
**Pull request by bcrochet**
Depends on openshift/mdns-publisher#14

Adds unit test target for mdns-publisher.

### CODE REVIEW

File: ci-operator/config/openshift/mdns-publisher/openshift-mdns-publisher-master.yaml [hunk 1]
```diff
@@ -34,3 +34,7 @@ tests:
   commands: make vet
   container:
     from: src
+- as: unit
+  commands: make test
+  container:
+    from: src
```
Comment (celebdor) [inline ci-operator/config/openshift/mdns-publisher/openshift-mdns-publisher-master.yaml]: can you fix the lack of newline at end of file?

**Comment by cybertron** [comment]
/test pj-rehearse

Dependency has merged, just making sure the job rehearsal still passes.

**Comment by cybertron** [comment]
And it did.

/lgtm

**Pull request merged**